index.html

```diff
@@ -91,8 +90,200 @@ <h1>I'm Rehema <br> Glad to see you !</h1>
+                      class="icon"
+                      alt="twitter-icon"
+                    />
+                  </a>
+                </li>
+                 <li>
+                  <a href="#" class="icon">
+                      <img class="linkedini"
+                      src="./images/1.svg"
+                      alt="twitter-icon"
+                    />
+                    <img class="linkedin"
+                      src="./images/n.svg"
+                      alt="twitter-icon"
+                    />
+                  </a>
+                </li>
+                <li>
+                  <a href="#">
+                    <img
+                      src="./images/medium.svg"
+                      class="icon m"
+                      alt="medium-icon"
+                    />
+                  </a>
+                </li>
+                <li>
+                  <a href="#">
+                    <img
+                      src="./images/github.svg"
+                      class="icon"
+                      alt="github-icon"
+                    />
+                  </a>
+                </li>
+                
+                <li>
+                  <a href="#">
+                    <img
+                      src="./images/angel.svg"
+                      class="icon"
+                      alt="angellist-icon"
+                    />
+                  </a>
+                </li>
+              </ul>
+            </div>
+            
+                <button type="submit">Get my resume</button>
+            </section>
+        </div>
+    
+        <!-- Languages Section -->
+        <div class="languages-section">
+            <section class="languages">
+                <div class="languages-title">
+                    <h2>Languages</h2>
+                    <img src="images/downarrow.svg" alt="">
+                </div>
+                
+                <div class="language-list">
+                    <div class="language">
+                        <img src="images/js-icon.svg" alt="" aria-hidden="true">
+                        <p>JavaSript</p>
+                    </div>
+                    <div class="language">
+                        <img src="images/html-icon.svg" alt="" aria-hidden="true">
+                        <p>HTML</p>
+                    </div>
+                    <div class="language">
+                        <img src="images/css-icon.svg" alt="" aria-hidden="true">
+                        <p>CSS</p>
+                    </div>
+                </div>
+            </section>
+     
+            <section class="frameworks">
+                <h2>Frameworks</h2>
+                <img src="images/next.png" alt="" aria-hidden="true">
+            </section>
+    
+            <section class="skills-a">
+                <h2>Skills</h2>
+                <img src="images/next.png" alt="" aria-hidden="true">
+            </section> 
+        </div>
+    </div>
```

Comment:
- [ ] In this case on line 250 please change `div` to `ul`, and then change all `section` child elements to `li` 🙌🏼